client/src/store/interfaces/authTypes.tsx

```diff
@@ -23,4 +31,23 @@ export interface LoginErrorAction {
 export interface LoginLoadingAction {
   type: "LOGIN_LOADING";
 }
-export type Action = LoginSuccessAction | LoginErrorAction | LoginLoadingAction;
+
+export interface RegisterSuccessAction {
+  type: "REGISTER_SUCCESS";
+  token: string;
+}
+export interface RegisterErrorAction {
+  type: "REGISTER_ERROR";
+  errors: [{ msg: string }];
+}
+export interface RegisterLoadingAction {
+  type: "REGISTER_LOADING";
+}
+
+export type Action =
```

Comment:
Maybe better if we divide this on LoginActions / RegisterActions

Reply:
Done